README.md

```diff
@@ -85,6 +85,16 @@ You should specify the inbound and outbound JSON mapping on your `RLMObject` sub
       };
     }
 
+JSON preprocessing can be done by implementing `jsonPreprocessing:` static method:
+
+```ObjC
+- (NSDictionary *)jsonPreprocessing:(NSDictionary *)dictionary {
```

Comment:
Documentation is wrong. This is not a static method and doesn't correspond to actual selector called [preprocessedJSON:](https://github.com/matthewcheok/Realm-JSON/pull/86/files#diff-01ee3b0e15a431be0882d21272698a29R164)